src/guide.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import io from 'socket.io-client';
+
+export default class GuidePanel extends React.Component {
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            video: null
+        }
+        this.client = io('http://localhost:9091');
```

Comment:
We should probably put the websocket client in its own class at some point, but this is probably fine for now